WeatherInfo/ViewControllers/Locations View Controller/LocationsViewController.swift

```diff
@@ -123,12 +123,15 @@ extension LocationsViewController: UITableViewDataSource {
         case .current:
             return 1
         case .favorite:
-            return max(favorites.count, 1)
+            return max(favorites.count, 0)
         }
     }
 
     func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
         guard let section = Section(rawValue: section) else { fatalError("Unexpected Section") }
```

Comment:
No fatal error, please.

Reply:
Right. I see some more fatalerrors. 
I will see to it we do not fatalError unless its crucial for experience. If a section header can not be loaded, we can just put it blank or a default header saying "Unkown section" or something.
Need to see where else all these fatalErrors are put in.